test/test_jit.py

```diff
@@ -6025,6 +6074,22 @@ def test_pow_int(x, y):
         self.checkScript(test_pow_float, (2.0, 2.0))
         self.checkScript(test_pow_int, (2.0, 2))
 
+        self.checkScript(test_isnan_int, (0,))
```

Comment:
We're mostly relying on these tests to make sure this stuff all works, the inputs should be more comprehensive like you had before

Reply:
This should be better now, i included a range of tests for + and - floats and ints.  Please reopen if there is an issue